src/pluto/grayscale_image.cr

```diff
@@ -5,11 +5,18 @@ class Pluto::GrayscaleImage < Pluto::Image
   property width : Int32
   property height : Int32
 
-  def self.new(red : Array(UInt8), green : Array(UInt8), blue : Array(UInt8), alpha : Array(UInt8), width : Int32, height : Int32)
+  def self.new(
+    red : Array(UInt8),
+    green : Array(UInt8),
+    blue : Array(UInt8),
+    alpha : Array(UInt8),
+    width : Int32,
+    height : Int32
+  )
     RGBAImage.new(red, green, blue, alpha, width, height).to_gray
   end
 
-  def initialize(@gray, @width, @height)
+  def initialize(@gray : Array(UInt8), @width : Int32, @height : Int32)
```

Comment:
What's the reasoning for putting the types back into the constructors if they're already defined with the `property` macro?

Reply:
Ah, right. I thought it wouldn't display in the API, but looks like it does.